hooks/queries/useVotingInfo.ts

```diff
@@ -1,17 +1,13 @@
-import { useQuery } from "@tanstack/react-query";
-import { defaultApy, voteTickerKey } from "constant";
+import { defaultApy } from "constant";
 import { getVotingInfo } from "queries";
+import useSWR from "swr";
 
 export function useVotingInfo() {
-  const queryResult = useQuery({
-    queryKey: [voteTickerKey],
-    queryFn: () => getVotingInfo(),
-    refetchInterval: 1000 * 60,
-    initialData: {
-      apy: defaultApy,
-      activeRequests: 0,
-      phase: "commit",
-    },
-  });
-  return queryResult;
+  const fallbackData = {
```

Comment:
since we don't need all of the react query features I switched to swr instead because its quite a bit smaller